leetcode/intervals.md

```diff
@@ -17,3 +17,30 @@ https://leetcode.com/problems/merge-intervals/
 ## Insert Interval

```

Comment:
Your Pull Request title: LEETCODE-insert-interval

Check result:

* Pull Request title must start with prefix in ['GENERATOR', 'LEETCODE']
* Pull Request title must contain group number in ['1021', '1022', '1013', '2021', '2022']
* Pull Request title action must start with ['Added', 'Deleted', 'Refactored', 'Moved', 'Fixed']